### PR TITLE
Update RELEASE_NOTES for 13.1.13 promotion

### PR DESCRIPTION
**Pull request by christopher-snapp**
### Description
The release notes needed to be updated to reflect the promotion of 13.1.13 to stable.

Signed-off-by: Christopher A. Snapp <csnapp@chef.io>

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -8,6 +8,49 @@ This document contains release notes for the current major release and all patch
...
+    - Switched compression from `bzip2` to `gzip`
+
+### Deprecation notices
+
+- SLES 11 is no longer supported per our [platform policy](https://docs.chef.io/platforms.html#platform-end-of-life-policy), as upstream support ended in March of this year.
+
+### Updates and Improvements
+
+- Yard 0.9.19 -> 0.9.20
+- Bundler 1.17.2 -> 1.17.2
```
Comment (tas50) [inline RELEASE_NOTES.md]: This seems wrong

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -8,6 +8,49 @@ This document contains release notes for the current major release and all patch
...
+- Postgres 9.6.10 -> 9.6.15
+- Chef v15.3.14 -> v15.4.45
```
Comment (tas50) [inline RELEASE_NOTES.md]: Chef Infra Client

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -8,6 +8,49 @@ This document contains release notes for the current major release and all patch
...
+- OpenResty 1.13.6.2 -> 1.15.8.1
+- Nokogiri 1.8.5 -> 1.10.4
+- Rebar3 -> 3.12.0
+- Updated erlang deps to be the latest
+- Loofah 2.2.3 -> 2.3.1
+- Erlang R18 -> 20.3.8.9
+- Updated for cookstyle
```
Comment (tas50) [inline RELEASE_NOTES.md]: Updated server setup cookbooks with Cookstyle fixes

**Comment by tas50** [comment]
openssl was probably also updated with a few CVEs patched. Those Ruby releases include a bunch of CVEs as well. Descriptions of some of that are in previous chef infra client release notes.

**Comment by christopher-snapp** [comment]
Not knowing any better I had told the docs team I'd update this PR when the web-docs PR was merged:

https://github.com/chef/chef-web-docs/pull/2204

I'm assuming any changes in this PR will require updates for `chef-web-docs` as well.

**Comment by PrajaktaPurohit** [comment]
@christopher-snapp thanks for pointing that out. I will issue a pr to chef-web-docs once this is merged.

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -8,6 +8,50 @@ This document contains release notes for the current major release and all patch
...
+    - Add elasticsearch info
+    - Switched compression from `bzip2` to `gzip`
+
+### Deprecation notices
+
+- SLES 11 is no longer supported per our [platform policy](https://docs.chef.io/platforms.html#platform-end-of-life-policy), as upstream support ended in March of this year.
+
+### Updates
+
+- Yard 0.9.19 -> 0.9.20
```
Comment (tas50) [inline RELEASE_NOTES.md]: We can probably skip this one. We're not even using the yard dep that comes in

**Pull request merged**